app/views/pages/_update_password_modal.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if user_signed_in? && current_user.password_needs_update? %>
+  <%= react_component('Modal', props: {
+    title: t('pages.home.update_pwd_modal.title'),
+    body: t('pages.home.update_pwd_modal.body', :update_password_link => link_to(t('pages.home.update_pwd_modal.update_link'), edit_user_registration_path)),
```

Comment:
Let's use colon notion instead of arrow notation i.e. `update_password_link: link_to(t('pages.home.update_pwd_modal.update_link'), edit_user_registration_path))`

Reply:
^ Please change this :)

Reply:
Oops forgot this one will update it.